### PR TITLE
Upgrade zendesk_apps_support.

### LINKED ISSUE #237: Issue with `nokogiri` version in lockfile
It looks like the current version of `nokogiri` listed in the lockfile is currently out of date and out of sync with the upstream [zendesk_apps_support](https://github.com/zendesk/zendesk_apps_support/) 

The version pointed to in this repo also has a [security vulnerability](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-9050), so it makes upgrading downstream projects to address that security vulnerability more difficult.

Thanks!

### PR DESCRIPTION
**Pull request by blimmer**
This resolves the nokogiri security vulnerability (Fixes #237)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -138,14 +138,14 @@ GEM
...
+    zendesk_apps_support (4.11.0)
       erubis
       i18n
       image_size
       jshintrb (~> 0.3.0)
       json
-      loofah
-      nokogiri (~> 1.6.8)
+      loofah (~> 2.2.1)
+      nokogiri (~> 1.8.1)
```
Comment (blimmer) [inline Gemfile.lock]: Both of these had security vulnerabilities.

File: .travis.yml [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 rvm:
-  - 2.0
   - 2.1
-  - 2.2.6
-  - 2.3.3
-  - 2.4.0
+  - 2.2
+  - 2.3
+  - 2.4
+  - 2.5
```
Comment (blimmer) [inline .travis.yml]: Added Ruby 2.5 since it is GA.

**Comment by blimmer** [comment]
cc @sandlerr @ocke who are active in this project - we'd really like to silence the CVE warnings on our apps that rely on this package. any feedback here? can we get this upgrade merged to mainline?

**Comment by hoangtrvu** [comment]
These dependencies have been bumped now so closing this PR.

**Pull request closed without merging**